libmpv/opengl_cb.h

```diff
@@ -163,10 +179,22 @@ extern "C" {
  *
  * glMPGetNativeDisplay("wl") should return a Wayland "struct wl_display *".
  *
- * glMPGetNativeDisplay("drm") should return a DRM FD casted to intptr_t (note
- * that a 0 FD is not supported - if this can happen in your case, you must
- * dup2() it to a non-0 FD).
- *
+ * glMPGetNativeDisplay("drm-params") should return an mp_drm_params structure pointer:
+ */
+struct mp_drm_params {
+    // DRM fd (int). Note that a 0 FD is not supported - if this
+    // can happen in your case, you must dup2() it to a non-0 FD).
+    int fd;
+
+    // currently used crtc id
+    int crtc_id;
+
+    // pointer to the drmAtomicRequest that is being used for the renderloop.
+    // This atomic request pointer should be usually created at every renderloop.
+    void *atomic_request;
```

Comment:
It's actually `drmModeAtomicReq` not `drmAtomicRequest`. Also I think you can change this to `struct _drmModeAtomicReq *atomic_request;`. I don't know why the header authors added this dumb `_` prefix - maybe it's meant to be private, but this is not a C convention, it's a violation of the C standard. So whatever.